src/api/spec/services/meta_controller_service/project_updater_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe ::MetaControllerService::ProjectUpdater do
+  let(:project) { create(:project, name: 'openSUSE_41') }
+  let(:admin_user) { create(:admin_user, login: 'Admin') }
+
+  before do
+    User.current = admin_user
+  end
+
+  context 'with valid meta' do
+    let(:meta) do
+      <<-HEREDOC
+      <project name="#{project.name}">
+      <title/><description/>
+      </project>
+      HEREDOC
+    end
+
+    let(:project_updater) { ::MetaControllerService::ProjectUpdater.new(project: project, request_data: Xmlhash.parse(meta)) }
+
+    describe '#new' do
+      it { expect(project_updater).not_to be_nil }
+    end
+
+    describe '#call' do
+      it { expect { project_updater.call }.not_to raise_error }
+    end
+
+    describe '#valid?' do
+      let(:project_updater_call) { project_updater.call }
+
+      it { expect(project_updater).to be_valid }
```

Comment:
This won't work because `project_updater_call` is never called. Just use `before` :wink:

Reply:
ops, done!